voter/models.py

```diff
@@ -241,13 +258,17 @@ def data_from_row(cls, row):
         row['registr_dt'] = str(row['registr_dt'])
         return row, {}
 
-    def build_current(self):
-        changelog = self.changelog.all()
+    def build_version(self, index):
+        changelog = self.changelog.all().order_by('snapshot_dt')
```

Comment:
Is this `order_by` necessary? I had assumed that the ordering specified in the ChangeTracker Meta class would apply here, but I'm not sure if that's true.